dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceFormatter.java

```diff
@@ -560,6 +893,29 @@ private final String generateObjectId(Object id, boolean fixedWidth) {
         return objId;
     }
```

Comment:
this generateObjectId should just call the new generateObjectId with the value from System.identityHashCode